canonical-web-and-design/cn.ubuntu.com
# add Kylin 16.04 announcement news article

### Done

Added a new news article
### QA

`make run`
Got to /news
See the new entry on the "ubuntu kylin" section
Go to the article by clicking the link
Check against: https://docs.google.com/document/d/1USCj-b3IxgAF3xNr9h-PYcOEJuh54aKe_o5ZPX7gVX8/edit?ts=570b926c#
